### DISCUSSION TITLE
Query on PrimeNG Angular Components

### DISCUSSION THREAD

**Discussion by deepthiseshabattar** [primefaces/community#390] [PrimeNG] [answered]
We would like to use the PrimeNG components for our product development , however before we use, we would like a few queries to be answered as our company is particular about regulatory compliances. and law
Any encryption  is used in PrimeNG Angular Components, if yes, pls share the key size, encryption type and intent for encryption - Authentication, data confidentiality, data integrity , data processing
Are there any export/re-export restrictions/exemptions/exceptions for PrimeNG
Based on law of the land, any export reference/clause for export of PrimeNG

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

We don't provide encryption, authentication, and data services. We do not collect data and process it. For further questions, please get in touch by contact@primetek.com.tr
